rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "stable"
+channel = "nightly"
```

Comment:
somehow I need to update to `nightly` so that one of the dependencies can be built. Not sure if this happens to others?

Reply:
Which dependency is that? Can we find another one, or use another version of it?

Reply:
seems in github action, it's ok. Maybe it's my local. I will revert this change.